src/I18n/Time.php

```diff
@@ -135,90 +135,8 @@ public function __construct($time = null, $tz = null)
         if (is_numeric($time)) {
             $time = '@' . $time;
         }
-        parent::__construct($time, $tz);
-    }
-
-    /**
-     * Returns a nicely formatted date string for this object.
-     *
-     * The format to be used is stored in the static property `Time::niceFormat`.
-     *
-     * @param \DateTimeZone|string|null $timezone Timezone string or DateTimeZone object
-     * in which the date will be displayed. The timezone stored for this object will not
-     * be changed.
-     * @param string|null $locale The locale name in which the date should be displayed (e.g. pt-BR)
-     * @return string Formatted date string
-     */
-    public function nice($timezone = null, $locale = null): string
```

Comment:
Are these methods on the `DateFormatTrait` already?

Reply:
Yes, or inherited from `Chronos`.